demisto_sdk/commands/pre_commit/hooks/sourcery.py

```diff
@@ -4,8 +4,11 @@
 from typing import Any, Dict
 
 from demisto_sdk.commands.common import tools
+from demisto_sdk.commands.common.content_constant_paths import CONTENT_PATH
 from demisto_sdk.commands.pre_commit.hooks.hook import Hook, join_files
 
+SOURCERY_CONFIG_PATH = CONTENT_PATH / ".sourcery.yaml"
```

Comment:
lets let the config argument and path come from args

Reply:
fixed